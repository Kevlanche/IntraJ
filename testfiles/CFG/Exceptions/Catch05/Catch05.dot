digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"}
{ rank = same; "List[7]"; "MethodDecl[12]"; "MethodDecl[76]"; "MemberClassDecl[88]"; "MemberClassDecl[96]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[16]"; "List[17]"; "List[18]"; "Opt[19]"; "Exit[20]"; "Entry[77]"; "Modifiers[78]"; "PrimitiveTypeAccess[80]"; "List[81]"; "List[82]"; "Opt[85]"; "Exit[86]"; "ClassDecl[89]"; "ClassDecl[97]"}
{ rank = same; "List[15]"; "Block[20]"; "List[79]"; "TypeAccess[83]"; "TypeAccess[84]"; "Block[86]"; "Modifiers[90]"; "Opt[92]"; "List[94]"; "List[95]"; "Modifiers[98]"; "Opt[100]"; "List[102]"; "List[103]"}
{ rank = same; "List[21]"; "List[87]"; "List[91]"; "TypeAccess[93]"; "List[99]"; "TypeAccess[101]"}
{ rank = same; "TryStmt[22]"; "VarDeclStmt[68]"}
{ rank = same; "Block[23]"; "List[50]"; "Opt[66]"; "UncheckedExceptions[67]"; "Modifiers[69]"; "PrimitiveTypeAccess[71]"; "List[72]"}
{ rank = same; "List[24]"; "BasicCatch[51]"; "List[70]"; "VariableDeclarator[73]"}
{ rank = same; "TryStmt[25]"; "ParameterDeclaration[52]"; "Block[56]"; "List[74]"; "Opt[75]"}
{ rank = same; "Block[26]"; "List[34]"; "Opt[48]"; "UncheckedExceptions[49]"; "Modifiers[53]"; "TypeAccess[55]"; "List[57]"}
{ rank = same; "List[27]"; "BasicCatch[35]"; "List[54]"; "VarDeclStmt[58]"}
{ rank = same; "ExprStmt[28]"; "ParameterDeclaration[36]"; "Block[40]"; "Modifiers[59]"; "PrimitiveTypeAccess[61]"; "List[62]"}
{ rank = same; "MethodAccess[29]"; "Modifiers[37]"; "TypeAccess[39]"; "List[41]"; "List[60]"; "VariableDeclarator[63]"}
{ rank = same; "List[31]"; "List[32]"; "List[33]"; "UncheckedExceptions[30]"; "List[38]"; "ExprStmt[42]"; "List[64]"; "Opt[65]"}
{ rank = same; "MethodAccess[43]"}
{ rank = same; "List[45]"; "List[46]"; "List[47]"; "UncheckedExceptions[44]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Catch05 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < test >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[16]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[17]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[18]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[19]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[20]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[21]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TryStmt[22]" [label="TryStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[23]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[24]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TryStmt[25]" [label="TryStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[26]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[27]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[28]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodAccess[29]" [label="MethodAccess\n < m >", fillcolor=white   style=filled  ]
"List[31]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[33]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[30]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"List[34]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"BasicCatch[35]" [label="BasicCatch\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[36]" [label="ParameterDeclaration\n < e >", fillcolor=white   style=filled  ]
"Modifiers[37]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[38]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[39]" [label="TypeAccess\n < TestSubException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[40]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[41]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[42]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodAccess[43]" [label="MethodAccess\n < m >", fillcolor=white   style=filled  ]
"List[45]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[46]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[47]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[44]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"Opt[48]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[49]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"List[50]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"BasicCatch[51]" [label="BasicCatch\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[52]" [label="ParameterDeclaration\n < e >", fillcolor=white   style=filled  ]
"Modifiers[53]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[54]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[55]" [label="TypeAccess\n < TestException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[56]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[57]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[58]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[59]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[60]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[61]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[62]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[63]" [label="VariableDeclarator\n < int a >", fillcolor=white   style=filled  ]
"List[64]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[65]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[66]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[67]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"VarDeclStmt[68]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[69]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[70]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[71]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[72]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[73]" [label="VariableDeclarator\n < int b >", fillcolor=white   style=filled  ]
"List[74]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[75]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[20]" [label="Exit\n ", fillcolor=white   style=filled  ]
"MethodDecl[76]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[77]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[78]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[79]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[80]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[81]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[82]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[83]" [label="TypeAccess\n < TestException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[84]" [label="TypeAccess\n < TestSubException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[85]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[86]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[87]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[86]" [label="Exit\n ", fillcolor=white   style=filled  ]
"MemberClassDecl[88]" [label="MemberClassDecl\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[89]" [label="ClassDecl\n < TestException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[90]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[91]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[92]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[93]" [label="TypeAccess\n < RuntimeException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[94]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[95]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MemberClassDecl[96]" [label="MemberClassDecl\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[97]" [label="ClassDecl\n < TestSubException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[98]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[99]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[100]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[101]" [label="TypeAccess\n < TestException >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[102]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[103]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[16]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[17]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[18]" [style=dashed, color=gray]
"MethodAccess[29]" -> "List[31]" [style=dashed, color=gray]
"MethodAccess[29]" -> "List[32]" [style=dashed, color=gray]
"MethodAccess[29]" -> "List[33]" [style=dashed, color=gray]
"MethodAccess[29]" -> "UncheckedExceptions[30]" [style=dashed, color=gray]
"ExprStmt[28]" -> "MethodAccess[29]" [style=dashed, color=gray]
"List[27]" -> "ExprStmt[28]" [style=dashed, color=gray]
"Block[26]" -> "List[27]" [style=dashed, color=gray]
"TryStmt[25]" -> "Block[26]" [style=dashed, color=gray]
"Modifiers[37]" -> "List[38]" [style=dashed, color=gray]
"ParameterDeclaration[36]" -> "Modifiers[37]" [style=dashed, color=gray]
"ParameterDeclaration[36]" -> "TypeAccess[39]" [style=dashed, color=gray]
"BasicCatch[35]" -> "ParameterDeclaration[36]" [style=dashed, color=gray]
"MethodAccess[43]" -> "List[45]" [style=dashed, color=gray]
"MethodAccess[43]" -> "List[46]" [style=dashed, color=gray]
"MethodAccess[43]" -> "List[47]" [style=dashed, color=gray]
"MethodAccess[43]" -> "UncheckedExceptions[44]" [style=dashed, color=gray]
"ExprStmt[42]" -> "MethodAccess[43]" [style=dashed, color=gray]
"List[41]" -> "ExprStmt[42]" [style=dashed, color=gray]
"Block[40]" -> "List[41]" [style=dashed, color=gray]
"BasicCatch[35]" -> "Block[40]" [style=dashed, color=gray]
"List[34]" -> "BasicCatch[35]" [style=dashed, color=gray]
"TryStmt[25]" -> "List[34]" [style=dashed, color=gray]
"TryStmt[25]" -> "Opt[48]" [style=dashed, color=gray]
"TryStmt[25]" -> "UncheckedExceptions[49]" [style=dashed, color=gray]
"List[24]" -> "TryStmt[25]" [style=dashed, color=gray]
"Block[23]" -> "List[24]" [style=dashed, color=gray]
"TryStmt[22]" -> "Block[23]" [style=dashed, color=gray]
"Modifiers[53]" -> "List[54]" [style=dashed, color=gray]
"ParameterDeclaration[52]" -> "Modifiers[53]" [style=dashed, color=gray]
"ParameterDeclaration[52]" -> "TypeAccess[55]" [style=dashed, color=gray]
"BasicCatch[51]" -> "ParameterDeclaration[52]" [style=dashed, color=gray]
"Modifiers[59]" -> "List[60]" [style=dashed, color=gray]
"VarDeclStmt[58]" -> "Modifiers[59]" [style=dashed, color=gray]
"VarDeclStmt[58]" -> "PrimitiveTypeAccess[61]" [style=dashed, color=gray]
"VariableDeclarator[63]" -> "List[64]" [style=dashed, color=gray]
"VariableDeclarator[63]" -> "Opt[65]" [style=dashed, color=gray]
"List[62]" -> "VariableDeclarator[63]" [style=dashed, color=gray]
"VarDeclStmt[58]" -> "List[62]" [style=dashed, color=gray]
"List[57]" -> "VarDeclStmt[58]" [style=dashed, color=gray]
"Block[56]" -> "List[57]" [style=dashed, color=gray]
"BasicCatch[51]" -> "Block[56]" [style=dashed, color=gray]
"List[50]" -> "BasicCatch[51]" [style=dashed, color=gray]
"TryStmt[22]" -> "List[50]" [style=dashed, color=gray]
"TryStmt[22]" -> "Opt[66]" [style=dashed, color=gray]
"TryStmt[22]" -> "UncheckedExceptions[67]" [style=dashed, color=gray]
"List[21]" -> "TryStmt[22]" [style=dashed, color=gray]
"Modifiers[69]" -> "List[70]" [style=dashed, color=gray]
"VarDeclStmt[68]" -> "Modifiers[69]" [style=dashed, color=gray]
"VarDeclStmt[68]" -> "PrimitiveTypeAccess[71]" [style=dashed, color=gray]
"VariableDeclarator[73]" -> "List[74]" [style=dashed, color=gray]
"VariableDeclarator[73]" -> "Opt[75]" [style=dashed, color=gray]
"List[72]" -> "VariableDeclarator[73]" [style=dashed, color=gray]
"VarDeclStmt[68]" -> "List[72]" [style=dashed, color=gray]
"List[21]" -> "VarDeclStmt[68]" [style=dashed, color=gray]
"Block[20]" -> "List[21]" [style=dashed, color=gray]
"Opt[19]" -> "Block[20]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[19]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[20]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"MethodDecl[76]" -> "Entry[77]" [style=dashed, color=gray]
"Modifiers[78]" -> "List[79]" [style=dashed, color=gray]
"MethodDecl[76]" -> "Modifiers[78]" [style=dashed, color=gray]
"MethodDecl[76]" -> "PrimitiveTypeAccess[80]" [style=dashed, color=gray]
"MethodDecl[76]" -> "List[81]" [style=dashed, color=gray]
"List[82]" -> "TypeAccess[83]" [style=dashed, color=gray]
"List[82]" -> "TypeAccess[84]" [style=dashed, color=gray]
"MethodDecl[76]" -> "List[82]" [style=dashed, color=gray]
"Block[86]" -> "List[87]" [style=dashed, color=gray]
"Opt[85]" -> "Block[86]" [style=dashed, color=gray]
"MethodDecl[76]" -> "Opt[85]" [style=dashed, color=gray]
"MethodDecl[76]" -> "Exit[86]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[76]" [style=dashed, color=gray]
"Modifiers[90]" -> "List[91]" [style=dashed, color=gray]
"ClassDecl[89]" -> "Modifiers[90]" [style=dashed, color=gray]
"Opt[92]" -> "TypeAccess[93]" [style=dashed, color=gray]
"ClassDecl[89]" -> "Opt[92]" [style=dashed, color=gray]
"ClassDecl[89]" -> "List[94]" [style=dashed, color=gray]
"ClassDecl[89]" -> "List[95]" [style=dashed, color=gray]
"MemberClassDecl[88]" -> "ClassDecl[89]" [style=dashed, color=gray]
"List[11]" -> "MemberClassDecl[88]" [style=dashed, color=gray]
"Modifiers[98]" -> "List[99]" [style=dashed, color=gray]
"ClassDecl[97]" -> "Modifiers[98]" [style=dashed, color=gray]
"Opt[100]" -> "TypeAccess[101]" [style=dashed, color=gray]
"ClassDecl[97]" -> "Opt[100]" [style=dashed, color=gray]
"ClassDecl[97]" -> "List[102]" [style=dashed, color=gray]
"ClassDecl[97]" -> "List[103]" [style=dashed, color=gray]
"MemberClassDecl[96]" -> "ClassDecl[97]" [style=dashed, color=gray]
"List[11]" -> "MemberClassDecl[96]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[20]" -> "UncheckedExceptions[67]" [color=red, constraint=false]
"Exit[20]" -> "UncheckedExceptions[49]" [color=red, constraint=false]
"Exit[20]" -> "UncheckedExceptions[30]" [color=red, constraint=false]
"Exit[20]" -> "UncheckedExceptions[44]" [color=red, constraint=false]
"Exit[20]" -> "VariableDeclarator[73]" [color=red, constraint=false]
"VariableDeclarator[73]" -> "Exit[20]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[73]" -> "UncheckedExceptions[67]" [color=red, constraint=false]
"VariableDeclarator[73]" -> "VariableDeclarator[63]" [color=red, constraint=false]
"UncheckedExceptions[67]" -> "VariableDeclarator[73]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[67]" -> "Exit[20]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[67]" -> "UncheckedExceptions[49]" [color=red, constraint=false]
"UncheckedExceptions[67]" -> "MethodAccess[43]" [color=red, constraint=false]
"UncheckedExceptions[49]" -> "UncheckedExceptions[67]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[49]" -> "Exit[20]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[49]" -> "MethodAccess[29]" [color=red, constraint=false]
"VariableDeclarator[63]" -> "VariableDeclarator[73]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[63]" -> "ParameterDeclaration[52]" [color=red, constraint=false]
"ParameterDeclaration[52]" -> "VariableDeclarator[63]" [color=blue, constraint=false, xlabel=" "] 
"ParameterDeclaration[52]" -> "UncheckedExceptions[30]" [color=red, constraint=false]
"ParameterDeclaration[52]" -> "UncheckedExceptions[44]" [color=red, constraint=false]
"UncheckedExceptions[44]" -> "ParameterDeclaration[52]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[44]" -> "Exit[20]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[44]" -> "MethodAccess[43]" [color=red, constraint=false]
"MethodAccess[43]" -> "UncheckedExceptions[67]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[43]" -> "UncheckedExceptions[44]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[43]" -> "ParameterDeclaration[36]" [color=red, constraint=false]
"ParameterDeclaration[36]" -> "MethodAccess[43]" [color=blue, constraint=false, xlabel=" "] 
"ParameterDeclaration[36]" -> "UncheckedExceptions[30]" [color=red, constraint=false]
"UncheckedExceptions[30]" -> "ParameterDeclaration[36]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[30]" -> "ParameterDeclaration[52]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[30]" -> "Exit[20]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[30]" -> "MethodAccess[29]" [color=red, constraint=false]
"MethodAccess[29]" -> "UncheckedExceptions[49]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[29]" -> "UncheckedExceptions[30]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[29]" -> "Entry[13]" [color=red, constraint=false]
"Entry[13]" -> "MethodAccess[29]" [color=blue, constraint=false, xlabel=" "] 
"Exit[86]" -> "Entry[77]" [color=red, constraint=false]
"Entry[77]" -> "Exit[86]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"MethodDecl[12]" -> "MethodDecl[76]" -> "MemberClassDecl[88]" -> "MemberClassDecl[96]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[16]" -> "List[17]" -> "List[18]" -> "Opt[19]" -> "Exit[20]" [style=invis]
"TryStmt[22]" -> "VarDeclStmt[68]" [style=invis]
"Block[23]" -> "List[50]" -> "Opt[66]" [style=invis]
"Block[26]" -> "List[34]" -> "Opt[48]" [style=invis]
"ParameterDeclaration[36]" -> "Block[40]" [style=invis]
"Modifiers[37]" -> "TypeAccess[39]" [style=invis]
"ParameterDeclaration[52]" -> "Block[56]" [style=invis]
"Modifiers[53]" -> "TypeAccess[55]" [style=invis]
"Modifiers[59]" -> "PrimitiveTypeAccess[61]" -> "List[62]" [style=invis]
"List[64]" -> "Opt[65]" [style=invis]
"Modifiers[69]" -> "PrimitiveTypeAccess[71]" -> "List[72]" [style=invis]
"List[74]" -> "Opt[75]" [style=invis]
"Entry[77]" -> "Modifiers[78]" -> "PrimitiveTypeAccess[80]" -> "List[81]" -> "List[82]" -> "Opt[85]" -> "Exit[86]" [style=invis]
"TypeAccess[83]" -> "TypeAccess[84]" [style=invis]
"Modifiers[90]" -> "Opt[92]" -> "List[94]" -> "List[95]" [style=invis]
"Modifiers[98]" -> "Opt[100]" -> "List[102]" -> "List[103]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
106", fillcolor=1]
Edges [label="Total Nr. Edges
143", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
13", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
19", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}