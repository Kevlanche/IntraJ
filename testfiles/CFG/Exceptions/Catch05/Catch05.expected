NodeName: Entry[13] Successors: [MethodAccess[29]]
NodeName: MethodAccess[29] Successors: [UncheckedExceptions[49], UncheckedExceptions[30]]
NodeName: UncheckedExceptions[49] Successors: [UncheckedExceptions[67], Exit[20]]
NodeName: UncheckedExceptions[67] Successors: [VariableDeclarator[73], Exit[20]]
NodeName: VariableDeclarator[73] Successors: [Exit[20]]
NodeName: Exit[20] Successors: []
NodeName: UncheckedExceptions[30] Successors: [ParameterDeclaration[36], ParameterDeclaration[52], Exit[20]]
NodeName: ParameterDeclaration[36] Successors: [MethodAccess[43]]
NodeName: MethodAccess[43] Successors: [UncheckedExceptions[67], UncheckedExceptions[44]]
NodeName: UncheckedExceptions[44] Successors: [ParameterDeclaration[52], Exit[20]]
NodeName: ParameterDeclaration[52] Successors: [VariableDeclarator[63]]
NodeName: VariableDeclarator[63] Successors: [VariableDeclarator[73]]
NodeName: Entry[77] Successors: [Exit[86]]
NodeName: Exit[86] Successors: []

NodeName: Exit[20] Predecessor: [UncheckedExceptions[67], UncheckedExceptions[49], UncheckedExceptions[30], UncheckedExceptions[44], VariableDeclarator[73]]
NodeName: UncheckedExceptions[67] Predecessor: [UncheckedExceptions[49], MethodAccess[43]]
NodeName: UncheckedExceptions[49] Predecessor: [MethodAccess[29]]
NodeName: MethodAccess[29] Predecessor: [Entry[13]]
NodeName: Entry[13] Predecessor: []
NodeName: MethodAccess[43] Predecessor: [ParameterDeclaration[36]]
NodeName: ParameterDeclaration[36] Predecessor: [UncheckedExceptions[30]]
NodeName: UncheckedExceptions[30] Predecessor: [MethodAccess[29]]
NodeName: UncheckedExceptions[44] Predecessor: [MethodAccess[43]]
NodeName: VariableDeclarator[73] Predecessor: [UncheckedExceptions[67], VariableDeclarator[63]]
NodeName: VariableDeclarator[63] Predecessor: [ParameterDeclaration[52]]
NodeName: ParameterDeclaration[52] Predecessor: [UncheckedExceptions[30], UncheckedExceptions[44]]
NodeName: Exit[86] Predecessor: [Entry[77]]
NodeName: Entry[77] Predecessor: []
